index.html

```diff
@@ -814,150 +814,156 @@ <h2>Privacy Considerations</h2>
 
     <section class="appendix">
       <h2>Test Vectors</h2>
-      <p>
-The following test vectors are provided to assist with implementers.
-      </p>
-
-      <pre class="example">
+      <section>
+        <h3>Representation: Ed25519Signature2020</h3>
+        <p>
+The signer needs to generate a private/public key pair with the private key used 
+for signing and the public key made available for verification. Below the 
+[[MULTIBASE]]/[[MULTICODEC]] representation for the public key, <code>ed25519-pub</code>,
+and the representation for the private key, <code>ed25519-priv</code>, are shown.
+        </p>
+        <pre class="example" title="Private and Public keys for Signature">
 {
-  "seed_0": "9b937b81322d816cfab9d5a3baacc9b2a5febe4b149f126b3630f93a29527017"
```

Comment:
We should probably have `@context` here and `"type": "Multikey"`.

Reply:
We should probably just break these values out. I'll do that after the merge. We haven't specified how / where to define `privateKeyMultibase` and we don't want that perma-thread holding up this PR.